README.md

```diff
@@ -1 +1,1380 @@
+
+There are certain similarities as well: the choice of scripting languages, ability to access the underlying Java functionality and the context data, logging methods, access to the request object, and ability to make back-channel HTTP requests—all converge into a similar experience at certain level. The script configuration and content for all three products could be represented as JSON and saved in the file system.
+
+The ability of scripts to communicate with external network resources is a powerful tool. The current security measures do not seem to apply restrictions to endpoints a script may access, which may represent an additional concern about the script's overall security if the administrative UI/API protections can be breached. However, because the same administrative privileges that allow manage scripts will likely allow to change other configuration, the network restriction policies may need to be implemented outside of the product configuration.
+
+Scripts add flexibility to the ForgeRock Identity Platform. While a script might not be performing at the same level as a native/standard implementation, custom scripts can be used to substitute functionality that is not yet present in the software or is specific to a certain deployment.
+
+## <a id="summary"></a>Summary
+
+[Back to the Top](#top)
+
+This section is a briefed overview of different scripting aspects in the three products.
+
+The [References](#references) section contains collection of links to the official scripting documentation. The links are organized by product and by area of concern.
+
+* ### <a id="summary-application-and-environment"></a>Application and Environment
+
+    Scripting application is augmenting or extending native functionality of a product. The extension points could be events associated with an object or a procedure; for example, an update event on a managed object in IDM. Or, the scripts could represent an essential part of a procedure, necessary for its successful completion; for example, scripts employed in AM authentication flows.
+
+    Scripts' environment could be described as access to methods and data. This may depend on the context the script is running in: the product and the particular procedure the script is extending. Even within the same product, the context may vary as the product components implement different functionality and expose different APIs.
+
+    ### <a id="summary-application-and-environment-am"></a> AM
+
+    Scripting application in AM could be summarized into the following categories:
+
+    * Authentication
+
+        * Modules and Chains
+            * Client-side
+            * Server-side
+        * Trees
+            * Client-side
+            * Server-side
+
+    * Authorization
+
+        * Scripting Policy Condition
+        * Access Token Modification
+
+    * Federation
+
+        * OIDC Claims Handling
```

Comment:
There is also an Access Token modification script

Reply:
Yes; does it not belong to Authorization?